tests/unit/modules/network/ios/test_ios_facts.py

```diff
@@ -72,7 +72,9 @@ def load_from_file(*args, **kwargs):
             output = list()
 
             for command in commands:
-                filename = str(command).split(" | ")[0].replace(" ", "_")
+                filename = (
+                    str(command).split(" | ", maxsplit=1)[0].replace(" ", "_")
```

Comment:
This will fail for Python 2.7.
```suggestion
                    str(command).split(" | ", 1)[0].replace(" ", "_")
```